our-work/collaborations/acbm.yml

```diff
@@ -0,0 +1,12 @@
+title: Serre Lab
+id: acbm
+description: Automated Continuous Behavioral Monitoring (ACBM) is a machine vision-based automated phenotyping method to detect and quantify the earliest-detected phenotypes in Tdp43Q331K knock-in mice.This method entails continuous video recording for 5 days that is then scored by a trained computer system. In addition to its sensitivity, objectivity and reproducibility, a major advantage of this method is the ability to acquire and archive video recordings and then analyze the data at sites remote from those of acquisition. CCV's collaboration with Serre Lab is to enable cloud based analysis of video recordings. Our goal is to establish sensitive, robust, objective, and open-source machine vision-based behavioral analysis tools that will be widely available to workers in the field.
```

Comment:
We are typically saying something about the lab first, then explaining our collaboration with the lab. In general the collaboration is not so much where we describe a project, but give a high-level description of the lab and the projects collaborate on with the.  In this case, so far we only have the ACBM project with Thomas, but if there were multiple projects, the description here should be inclusive of all of them. So for now ACBM is an example, but don't focus all the text on the details. If there is software that is public for ACBM, then the software section is more 1:1 with projects, that that's where the details would go

Reply:
Thank you so much for this clarification. I was confused about what goes where. Now the description should give a more general idea of what lab does and ACBM software is added to the software section. We don't have public GitHub yet. But I will update it when we have it.